src/fetching/tasks/patchTaskItem.ts

```diff
@@ -0,0 +1,23 @@
+import { defaultFetch } from '..';
+import { ZetkinSmartSearchFilter, ZetkinTask } from '../../types/zetkin';
+
+const patchTaskItem = (orgId : string, taskId: string, fetch = defaultFetch) => {
+    return async (data: ZetkinSmartSearchFilter[]):Promise<ZetkinTask> => {
+        const url = `/orgs/${orgId}/tasks/${taskId}/target`;
+        const body = {
+            filter_spec: data,
+        };
+
+        const res = await fetch(url, {
+            body: JSON.stringify(body),
+            headers: {
+                'Content-Type': 'application/json',
+            },
+            method: 'PATCH',
+        });
+        const resData = await res.json();
+        return resData?.data;
+    };
+};
+
+export default patchTaskItem;
```

Comment:
This module isn't really for patching a task, but for patching a task's target group. I think renaming it is wise, also because it will otherwise collide with what River is doing (even though it's not strictly related).

If you know the ID of the Smart Search (which I believe is included in the task data) you can in fact patch the Smart Search query at `/orgs/{org_id}/people/queries/{query_id}` and use the same "fetch" function for any query, be it a standalone, call assignment or task query.